components/project-sharing.tsx

```
@@ -38,7 +39,26 @@ interface ProjectSharingProps {
+  useEffect(() => {
+    setIsClient(true)
+  }, [])
+
+  // Handle authentication based on bypass mode - only get user after client mount
+  let user = null
+  if (isClient) {
+    if (BYPASS_CLERK) {
+      user = { id: 'dev-user' }
+    } else {
+      // eslint-disable-next-line @typescript-eslint/no-var-requires
+      const { useUser } = require("@clerk/nextjs")
```

ESLint

[error] A `require()` style import is forbidden.

(@typescript-eslint/no-require-imports)